src/main/java/us/lsi/dp1/newcorporder/match/company/CompanyTile.java

```diff
@@ -11,26 +12,28 @@ public class CompanyTile {
 
     private final Company company;
 
+    @Setter
     private Conglomerate currentConglomerate;
+    @Setter
```

Comment:
¿Dónde estáis llamando a los setters desde fuera de la clase? Lo mejor es modificarlos en los métodos de la misma clase para evitar exponerlos y que se puedan modificar sin control.